installer/installer.spec

```diff
@@ -70,7 +70,7 @@ exe_installer = EXE(
     [],
     exclude_binaries=True,
     name='DXRandoInstaller',
-    debug=False,
+    debug=True,
```

Comment:
this debug bool, what exactly does it do?

Reply:
the bingo exe isn't using it

Reply:
I think it turned on the installer warnings that get generated into that file.  Turning it on also magically resolved whatever missing package issue I was having when running on a freshly installed VM